compaction/compaction_manager.cc

```diff
@@ -340,10 +340,10 @@ future<sstables::compaction_result> compaction_manager::task::compact_sstables(s
 
     co_return co_await sstables::compact_sstables(std::move(descriptor), cdata, t.as_table_state());
 }
-future<> compaction_manager::task::update_history(replica::table& t, const sstables::compaction_result& res, const sstables::compaction_data& cdata) {
+future<> compaction_manager::task::update_history(compaction::table_state& t, const sstables::compaction_result& res, const sstables::compaction_data& cdata) {
```

Comment:
the `compaction::` namespace prefix is probably not needed as compaction.hh is `using namespace compaction`